edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Service/modules/CommonModule.cs

```diff
@@ -386,5 +409,13 @@ async Task<IAuthenticator> GetCloudTokenAuthenticator(IComponentContext context)
 
             return tokenAuthenticator;
         }
+
+        private Option<int> ExtractCheckFrequency(StoreAndForwardConfiguration storeAndForwardConfiguration)
+        {
+            long defaultMaxStorageSize = long.MaxValue;
+            int defaultCheckFrequency = 120;
+            StoreLimits storeLimits = storeAndForwardConfiguration.StoreLimits.GetOrElse(() => new StoreLimits(defaultMaxStorageSize, Option.Some(defaultCheckFrequency)));
```

Comment:
>GetOrElse [](start = 79, length = 9)

Use .Map here and within the delegate inside it, just use GetOrElse on the argument of the delegate to get the value of CheckFrequency or return the default

Reply:
This is irrelevant now that it has been changed - since the checker won't start until the twin updates occur, we don't need to give it any configuration from env variables.